backend/geonature/core/gn_synthese/synthese_config.py

```diff
@@ -99,3 +99,5 @@
     {"prop": "dataset_name", "name": "JDD", "max_width": 200},
     {"prop": "observers", "name": "observateur", "max_width": 200},
 ]
+
+MODULE_CODE = "synthese"
```

Comment:
La config du module_code se répercute partout ?

Reply:
J'ai changé la manière dutiliser le module_code . En appelant ModuleService et en utilisnt currentModule

Reply:
Je n'y connais rien techniquement, mais je m'interroge que l'on ajoute un paramètre ``MODULE_CODE`` pour la synthèse dans la configuration de GeoNature, alors que l'on n'a pas ce fonctionnement dans les autres modules, où le module_code n'est pas géré ni modifiable dans la configuration du module. 
Je m'interroge donc sur l'hétérogénéité de fonctionnement et les éventuelles conséquences.

Reply:
> Je n'y connais rien techniquement, mais je m'interroge que l'on ajoute un paramètre `MODULE_CODE` pour la synthèse dans la configuration de GeoNature, alors que l'on n'a pas ce fonctionnement dans les autres modules, où le module_code n'est pas géré ni modifiable dans la configuration du module. Je m'interroge donc sur l'hétérogénéité de fonctionnement et les éventuelles conséquences.

C'est ce que je dis dans le précédent commentaire, j'ai changé la manière d'utiliser le module_code . Il faut que j'enleve à ce commit : https://github.com/PnX-SI/GeoNature/pull/2910/commits/30bbe6f391345d1f07709006800f34efc94d6d5f  l'ajout que j'ai fait au tout début du MODULE_CODE